src/main/resources/assets/lambdamap/lang/en_us.json

```diff
@@ -13,6 +13,11 @@
   "lambdamap.marker.new.player_pos": "Player Pos",
   "lambdamap.marker.new.x": "X:",
   "lambdamap.marker.new.z": "Z:",
+  "lambdamap.marker.confirm_deletion.prompt": "Are you sure you want to %s %s?",
+  "lambdamap.marker.confirm_deletion.prompt.action": "permanently delete",
+  "lambdamap.marker.confirm_deletion.prompt.unnamed": "Unnamed Marker",
+  "lambdamap.marker.confirm_deletion.confirm": "I'm sure",
```

Comment:
If vanilla has those translation strings already, use them instead to have better localization, and lessen translation burden in the mod.

Reply:
Cancel is of course present in the vanilla files, the other ones however aren't.
I could change the prompt to say "Delete" instead of "I'm sure" but I can't get rid of the marker prompt translation as the version used for world deletion included in the lang file of course includes the word "world".

Reply:
What about `ScreenTexts.PROCEED` and `ScreenTexts.CANCEL`?

Reply:
![image](https://user-images.githubusercontent.com/27809595/147882825-2705ae93-471a-4800-8826-6f591f27756d.png)
Version using `ScreenTexts.PROCEED` instead of "I'm sure"

Reply:
Seems fine to me, I think it's better.